libucontext.yaml

```
@@ -0,0 +1,44 @@
+package:
+  name: libucontext
+  version: 1.2
+  epoch: 0
+  description: "ucontext function implementations"
+  copyright:
+    - license: "ISC"
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - cmake
+      - build-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kaniini/libucontext
+      tag: v${{package.version}}
+      expected-commit: 4dde3417b4bb4b1b1545bd913be337680b5e28c3
+
+  - runs: |
+      make
+      make install
```

```suggestion
      make DESTDIR="${{targets.destdir}}" install
```
I think we may need this else we won't get the binary in the end apk package.   It looks like `DESTDIR` is supported https://github.com/kaniini/libucontext/blob/be80075e957c4a61a6415c280802fea9001201a2/Makefile#L210C41-L210C48

---

done